<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultyWeaponsPage</class>
 <widget class="QWidget" name="DifficultyWeaponsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="RealisticGunnery">
     <property name="text">
      <string>Realistic gunnery</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="LimitedAmmo">
     <property name="text">
      <string>Limited ammo</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="LimitedFuel">
     <property name="text">
      <string>Limited fuel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="BombFuzes">
     <property name="text">
      <string>Bomb fuzes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RealisticTorpedoing">
     <property name="text">
      <string>Realistic torpedoing</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="RealisticMissilesVariation">
     <property name="text">
      <string>Realistic missiles variation</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
